tiledb/sm/serialization/tiledb-rest.capnp

```diff
@@ -1253,3 +1257,42 @@ struct LoadArraySchemaResponse {
   schema @0 :ArraySchema;
   # The loaded ArraySchema
 }
+
+struct QueryChannel {
+  # structure representing a query channel, that is a
+  # stream of data within a TileDB query. Such channels can be generated
+  # for the purpose of avoiding processing result items multiple times
+  # in more complex queries such as e.g. grouping queries.
+
+  default @0 :Bool;
+  # True if a channel is the default query channel
+
+  segments @1 :List(ChannelSegment);
+  # contains the list of segments in a query channel.
+  # If the channel is the default query channel, this list contains
+  # a single segment.
+}
+
+struct ChannelSegment {
```

Comment:
We don't need this... Segments will be represented in the buffers. The QueryChannel struct above should have the list of aggregates defined here, and later a grouping operation when we allow it.